examples/tv-app/android/App/platform-app/src/main/java/com/matter/tv/server/MatterCommissioningPrompter.java

```diff
@@ -144,4 +155,43 @@ private void createNotificationChannel() {
       notificationManager.createNotificationChannel(channel);
     }
   }
+
+  static class VH extends Handler{
+
+    private final WeakReference<MatterCommissioningPrompter> mPrompterWeakReference;
+    public VH(MatterCommissioningPrompter commissioningPrompter){
+      super(Looper.getMainLooper());
+      mPrompterWeakReference = new WeakReference<>(commissioningPrompter);
+    }
+    @Override
+    public void handleMessage(@NonNull Message msg) {
+      MatterCommissioningPrompter commissioningPrompter = mPrompterWeakReference.get();
```

Comment:
arg1 and arg2 seem unused here. Is there a plan to use them? If not, maybe don't need to set them.